main.lua

```diff
@@ -89,6 +71,8 @@ function startServer(bufpane, argsUserdata)
         table.insert(args, a)
     end
 
+    local _ = table.remove(args, 1) -- ignore `start` argument
+
```

Comment:
I don't like handling this here. The "start" argument should be removed before calling this function (same goes for the other subcommands too).